projects/go-lib/src/lib/components/go-off-canvas/go-off-canvas.component.html

```diff
@@ -11,13 +11,8 @@
   role="dialog"
 >
   <div class="go-off-canvas__header">
-    <button
-      (click)="closeOffCanvas()"
-      class="go-off-canvas__button"
-      type="button"
-    >
-      Close
-    </button>
+    <h2>{{ header }}</h2>
```

Comment:
@deotte what happens when someone doesn't pass in a header?

Reply:
You are right. I will change that over to an `*ngIf`.